packages/core/src/utils/StringBuilder.ts

```diff
@@ -23,7 +23,7 @@ export default class StringBuilder {
   public static concat(...builders: StringBuilder[]): string {
     return builders
       .map((b) => b.toString())
-      .filter(Boolean)
```

Comment:
I actually like the `filter(Boolean)`. It's quite common in my day-to-day job. :)

Reply:
Ok :)